prime-router/src/test/kotlin/azure/AuthenticationTests.kt

```diff
@@ -5,26 +5,35 @@ import kotlin.test.assertFalse
 import kotlin.test.assertTrue
 
 class AuthenticationTests {
-    val verifier = OktaAuthenticationVerifier()
+    private val verifier = OktaAuthenticationVerifier()
 
     @Test
     fun `test user level checkMembership`() {
         val userMemberships = listOf("DHpima_az_phd")
-        assertTrue(verifier.checkMembership(userMemberships, PrincipalLevel.USER, "pima_az_phd"))
-        assertFalse(verifier.checkMembership(userMemberships, PrincipalLevel.USER, "foo"))
+        assertTrue(verifier.checkMembership(userMemberships, PrincipalLevel.USER, organizationName = "pima-az-phd"))
```

Comment:
You can see the bug here. Our orgNames use `-` for spacing. Our Okta groups use `_` for spaces.